docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.8'
+version: "3.8"
```

Comment:
this is not a `**/*.js|md` file why had it been formatted?

Reply:
agree with @atxr

Reply:
Seems good!